xl_auth/permission/forms.py

```diff
@@ -137,6 +151,23 @@ def validate_collection_id(self, field):
             raise ValidationError(_('Collection ID "%(collection_id)s" does not exist',
                                     collection_id=field.data))
 
+    def validate_global_registrant(self, field):
+        """Validate that user is admin if global_registrant is changed."""
+        if self.current_user.is_admin:
+            return
+
+        target_permission = Permission.get_by_id(self.target_permission_id)
+        if not target_permission:
+            return  # handled elsewhere
+
+        if target_permission.global_registrant != field.data:
+            raise ValidationError(_('You do not have sufficient privileges '
+                                    'for this operation.'))
+
+        if field.data and (target_permission.user_id != self.user_id.data):
+            raise ValidationError(_('Cannot move Global Registrant to other '
```

Comment:
The permission isn't visible to non-admins, right? I think it would be less confusing for the users to use the error message from above here as well.

Reply:
I was thinking it would be less confusing to get another error message for something that should seemingly work from the users perspective. But perhaps you are right.

Reply:
Changed